avoid that AutoMap tries to create a log file in installation directory,
log to current directory instead using PID of current process
author: Kenneth Hoste (HPC-UGent)
--- AutoMap_v1.0.sh.orig	2020-04-16 14:34:15.327400539 +0200
+++ AutoMap_v1.0.sh	2020-04-16 14:33:57.979381000 +0200
@@ -241,7 +241,7 @@
     vcf=${vcfs[$k]}
     id=${ids[$k]}
 
-    here="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+    here="$PWD/AutoMap_pid$$"
 
     nbvar=$(grep -v "#" $vcf | grep -P "AD|DP4" | grep GT | wc -l)
 
